release_notes/index.adoc

```diff
@@ -20,3 +20,11 @@ technical notes, see the
 https://docs.openshift.com/enterprise/3.1/release_notes/index.html[OpenShift
 Enterprise 3.1 Release Notes].
 endif::[]
+
+[[release-versioning-policy]]
+== Versioning Policy
+{product-title} does not have non-backwards compatible changes and, therefore,
+does not drop features on versions earlier than v3. Some upgrades may require
+changes by the administrator. Clients will continue to work against major
+version servers. For example, a 3.1 client will work against 3.0, 3.1, and 3.2.
+Not all features are available in 3.1 clients against 3.2 servers.
```

Comment:
What exactly in the text is not clear? What needs further elaboration?

On Aug 8, 2016 6:22 PM, "Aleksandar Kostadinov" <notifications@github.com>
wrote:

> In release_notes/index.adoc
> <https://github.com/openshift/openshift-docs/pull/2496#discussion_r73966955>
> :
>
> > @@ -20,3 +20,11 @@ technical notes, see the
> >  https://docs.openshift.com/enterprise/3.1/release_notes/index.html[OpenShift
> >  Enterprise 3.1 Release Notes].
> >  endif::[]
> > +
> > +[[release-versioning-policy]]
> > +== Versioning Policy
> > +{product-title} does not have non-backwards compatible changes and, therefore,
> > +does not drop features on versions earlier than v3. Some upgrades may require
> > +changes by the administrator. Clients will continue to work against major
> > +version servers. For example, a 3.1 client will work against 3.0, 3.1, and 3.2.
> > +Not all features are available in 3.1 clients against 3.2 servers.
>
> Makes sense, IMO good to elaborate on that in the text as it is not clear
> for me.
>
> —
> You are receiving this because you were mentioned.
> Reply to this email directly, view it on GitHub
> <https://github.com/openshift/openshift-docs/pull/2496/files/96bec963399fd1698069a0948690bca7e9c23aea#r73966955>,
> or mute the thread
> <https://github.com/notifications/unsubscribe-auth/ABUZmuGKESVKIasSVpASoUfT3xMxUjR9ks5qd6wagaJpZM4JLgeb>
> .
>